### DISCUSSION TITLE
Problems calibrating the Tree

### DISCUSSION THREAD

**Discussion by ArrX8** [Lailloken/Exile-UI#279] [General]
Hello, I have tried to calibrate the tree in many ways but I can't. Sometimes it tests positive and works, but this happens for 1-2 seconds.
I don't know if it will have to see that I use windows 11. In the last league with windows 10 it worked fine.
Video:
https://github.com/Lailloken/Lailloken-UI/assets/58579952/c52ccbba-c150-4fad-9421-620164d0af44

Thanks in advance and forgive my English.

**Comment by Lailloken** [comment]
Do you have Auto-HDR enabled (or some other graphics filter, such as ReShade or Nvidia GFE)? That would explain why it switches between positive and negative within seconds -- something may be altering the color-values on the screen.

**Comment by ArrX8** [comment]
Thank you very much for such a quick reply. 

I couldn't tell you if I use it. My card is Nvidia.
I would not know how to look at what option it is to see if it is enabled or not.

**Reply by Lailloken** [reply]
You can first search Google for Windows 11 Auto HDR, I'm pretty confident it's the issue here since you said it worked fine before on Win 10. Try to find how to enable/disable it and then check if it's enabled on your system.

**Reply by ArrX8** [reply]
If this is it, I found it but I thought it would be an option within the Nvidia control panel

![image](https://github.com/Lailloken/Lailloken-UI/assets/58579952/3c497dc8-e58a-42c6-8234-df02d9e5ff8b)

**Comment by ArrX8** [comment]
I have changed the game to DirectX12 (beta), it never worked very well for me... but the tool works.

**Reply by Lailloken** [reply]
That's strange, what was it set to before?

**Reply by ArrX8** [reply]
Vulkan, although I also had a problem with Directx11 and it didn't work on either of them.

When I press the middle key of the mouse the screen flashes once, whereas with DirectX12 it doesn't.
